resources/locales.clj

```diff
@@ -1,5 +1,5 @@
 {
-  :locales  #{"en" "zh" "zh-HK" "zh-TW" "uk" "fr" "nl" "tr" "sk" "it" "es" "nb" "ca" "de" "fa" "ja" "vi" "pl" "pt" "ru" }
+  :locales  #{"en" "nl" "zh" "zh-HK" "fr" "ja" "de" "es" "ca" "pl" "zh-TW" "it" "ru" "fa" "sk" "pt" "tr" }
```

Comment:
Is this related to this change?

Reply:
Running the build script ends up changing `locales.clj`. Commiting this was accidental. Will revert